packages/zone.js/test/common/promise-disable-wrap-uncaught-promise-rejection.spec.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+class TestRejection {
+  prop1?: string;
+  prop2?: string;
+}
+
+describe('disable wrap uncaught promise rejection', () => {
+  it('should notify Zone.onHandleError if no one catches promise', (done) => {
```

Comment:
```suggestion
  it('should notify Zone.onHandleError if promise is uncaught', (done) => {
```

Is `uncaught` the right word? you mean no one processes rejection??? Or is that the same thing?

Reply:
I believe `uncaught` and `no one processes rejection` is the same thing here.